### DISCUSSION TITLE
8.6.7 Release

### DISCUSSION THREAD

**Discussion by Donnnno** [Arcticons-Team/Arcticons#1949] [Announcements]
###### Released: Jan 25, 2024
* 🎉 380 new and updated icons! (thanks @Kaiserdragon2, @AxFo, @Elaborendum, @Ayecptn, @SagXD, @Dgcampbe, @Npnpatidar, @TotallyAvailable, @Alchemiker, @Dumdus, @Dioxias, @Dazewell, @rahulTPatil, @JRBros2346, @Plasmid-zz, @KR4U7H3R, @ThatBlueEnder, @Sande-Santhosh, @MikeTheGreat, @Andrew-aiken, @LukeVader-IV, @Gaenseblume, @DeltriDev, @Petlyh )
* 📁 New search categories for Google and Microsoft apps, also some emojis are added.
* ✨ Updates to make sure many apps work again.
* 🚀 Dashboard updates.
* 🔥 8673 icons in total!

<hr /><em>This discussion was created from the release <a href='https://github.com/Arcticons-Team/Arcticons/releases/tag/8.6.7'>8.6.7 Release</a>.</em>

**Comment by Kaiserdragon2** [comment]
You mentioned me wrong

**Reply by Donnnno** [reply]
Yes, I noticed it this morning, should be fixed now! :)

**Reply by Kaiserdragon2** [reply]
No worries it's fixed
